pkg/logging/logging.go

```diff
@@ -207,7 +208,7 @@ func loggingProcessAdapter(ctx context.Context, driver Driver, dataStore, hostAd
 		buf := make([]byte, 32<<10)
 		_, err := io.CopyBuffer(writer, reader, buf)
 		if err != nil {
-			logrus.Errorf("failed to copy stream: %s", err)
+			log.L.Errorf("failed to copy stream: %v", err)
```

Comment:
This should meld into the carried commit so that the carried commit is still compilable